client/src/app/pages/absences/absences.page.html

```diff
@@ -11,17 +14,17 @@
     </ion-label>	
   </ion-item>	
   <ion-list>	
-      <ion-card *ngFor='let falta of faltas'>	
+      <ion-card *ngFor='let absence of absences'>	
         <ion-card-header>	
           <ion-card-subtitle>	
-            {{falta.lesson.turma.course}} - {{falta.lesson.turma.title}}	
+            {{absence.lesson.turma.course}} - {{absence.lesson.turma.title}}	
           </ion-card-subtitle>	
           <ion-card-title>	
-            {{falta.lesson.date}}	
+            {{absence.lesson.date}}	
           </ion-card-title>	
         </ion-card-header>	
         <ion-card-content>	
-          Semestre {{falta.lesson.turma.semester}} de {{falta.lesson.turma.year}}	
+          Semestre {{absence.lesson.turma.semester}} de {{absence.lesson.turma.year}}	
```

Comment:
The hard-coded 'Semestre' will not work nicely with the Semester enum in django... it will probably be necessary to change this structure to {{absence.lesson.classGroup.semester.name}} de {{absence.lesson.classGroup.year}}

Reply:
Also.... remove Portuguese from code kkk